app/assets/scripts/actions/base-data.js

```diff
@@ -80,5 +82,7 @@ export function fetchBaseData() {
     fetcher('countries');
     fetcher('sources');
     fetcher('parameters');
+    fetcher('manufacturers');
```

Comment:
Why did you decide to add these to the base data?

Personally, I am trying to eliminate the base data / base stats concept, as these requests happen on every page, even the static about page, where we do not at all need this data. Would it be reasonable enough to change the code to only fetch the lists when we need them? Maybe in a useEffect within those components?

Reply:
Honestly I put it here to maintain consistency with most of the requests that are using the reduxeed api format (fetching, fetched etc). Can add it here as a one of useEffect request but I thought maybe this data was needed elsewhere and I thought it was simpler to add here than to setup a single action/reducer.
As of now the app calls `fetchBaseData` in main.js. We could refactor this call to take params, and introduce it to relevant components. perhaps in separate PR? what do you think?

Reply:
The general idea was to move away from redux, we have useEffect fetches in other places right now. I'll leave it to you to decide if that blows the scope of this implementation.